src/util.h

```diff
@@ -202,6 +202,7 @@ class ArgsManager
     void ParseParameters(int argc, const char*const argv[]);
     void ReadConfigFile(const std::string& confPath);
     std::vector<std::string> GetArgs(const std::string& strArg);
+    std::vector<std::string> GetUniqueArgs(const std::string& strArg);
```

Comment:
Is this use-case common enough to add a new argument getter to the API for?
If not, I'd prefer chaining more general functions, for example defining a function `std::vector<string> UniqueStrings(const std::vector<string>&)` then using it as
```c++
UniqueStrings(GetArgs("wallet"))
```